avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -263,11 +270,16 @@ public Object convert(Schema schema, Object value) {
 
   private Cache<Schema, org.apache.avro.Schema> fromConnectSchemaCache;
   private Cache<org.apache.avro.Schema, Schema> toConnectSchemaCache;
-
+  private boolean includeConnectMeta;
 
   public AvroData(int cacheSize) {
+    this(cacheSize, true);
+  }
+
+  public AvroData(int cacheSize, boolean includeConnectMeta) {
```

Comment:
Interesting one, 

We actually were thinking this would be useful, as now we have Avro In Schema = Avro Out Schema where a flow is simply Avro In (kafka) -> Avro Out (hdfs) there should be technically no need for any additional connect meta data, and actually we may prefer to honour keeping the schema intact without modification. 

Saying that not heavily opposed to making it private if you feel strongly on this, just as you've stated it can be re-used by others and isn't that useful?